src/components/dashboard/main-content/course-enrollments/course-cards/BaseCourseCard.jsx

```diff
@@ -91,51 +92,6 @@ class BaseCourseCard extends Component {
     return [...firstMenuItems, ...lastMenuItems];
   };
 
-  getDateMessage = () => {
-    const { type, pacing, endDate } = this.props;
-    const formattedEndDate = endDate ? dayjs(endDate).format('MMMM D, YYYY') : null;
-    let message = '';
-    if (formattedEndDate) {
-      switch (type) {
-        case COURSE_STATUSES.inProgress: {
-          if (pacing === 'self') {
-            message += `Complete at your own speed before ${formattedEndDate}.`;
```

Comment:
[clarification/confirmation] Is removing this messaging intended? Frames in the ["Delivery" ](https://www.figma.com/file/8da9TyEpEzgxQld3DZE67w/LC2---Top-down-Assignment?type=design&node-id=2616-102193&mode=dev) section of the Figma mocks still show the same messaging of "Complete at your own speed..." for self-paced Open Courses. Seems like this would be worth checking in about.

Example:

<img width="827" alt="image" src="https://github.com/openedx/frontend-app-learner-portal-enterprise/assets/2828721/852e4d30-f606-446b-a9f7-a80916732989">

Reply:
@adamstankiewicz I must have overlooked it. Added the "**Self-paced**" message as per figma designs. 
I have requested Alena from UX to provide the Self-paced redirect link.

Reply:
Alena got back to me a few minutes ago to confirm what you have is correct. Appreciate the patience to clarify the conflicting mocks 😄